### PR TITLE
Improves admin verb logging

### PR DESCRIPTION
**Pull request by ChangelingRain**
Splits logging for player preferences out of admin verb logging, and splits admin toggles out of admin verbs.

### CODE REVIEW

**Comment by ChangelingRain** [comment]
For reference as to why, please look at this; https://atlantaned.space/newSS13tools/stats/monthlyStats.php?year=2017&month=03&stat=admin_verb

**Comment by optimumtact** [comment]
@nfreader

File: code/modules/admin/admin_verbs.dm [hunk 1]
```diff
@@ -484,7 +479,7 @@ GLOBAL_LIST_INIT(admin_verbs_hideable, AVerbsHideable())
 				mob.mouse_opacity = 0
 		log_admin("[key_name(usr)] has turned stealth mode [holder.fakekey ? "ON" : "OFF"]")
 		message_admins("[key_name_admin(usr)] has turned stealth mode [holder.fakekey ? "ON" : "OFF"]")
-	feedback_add_details("admin_verb","SM") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
+	feedback_add_details("admin_verb","Stealth Mode") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```
Comment (Iamgoofball) [inline code/modules/admin/admin_verbs.dm]: check with @atlanta-ned about if this actually works

**Comment by nfreader** [comment]
Hooray! Why were they shortened in the first place though?

**Comment by Cyberboss** [comment]
Because the first person to make them set a bad fucking example

**Comment by optimumtact** [comment]
feature freeze

**Comment by ChangelingRain** [comment]
this is not a feature

**Comment by Cyberboss** [comment]
Not a feature

**Comment by Shadowlight213** [comment]
Logging for GLOB when

**Comment by Niknakflak** [comment]
oranges is in a mood

**Comment by nfreader** [comment]
Just a heads up, `admin_verb` will probably show both the shorthand and the longer updated names for the April stats. So that'll be annoying, but something we'll have to deal with.

**Comment by lzimann** [comment]
It's for the best.

**Pull request merged**